tools/exchange/src/main/scala/com/vesoft/nebula/tools/importer/config/SinkConfigs.scala

```diff
@@ -0,0 +1,47 @@
+/* Copyright (c) 2020 vesoft inc. All rights reserved.
+ *
+ * This source code is licensed under Apache 2.0 License,
+ * attached with Common Clause Condition 1.0, found in the LICENSES directory.
+ */
+
+package com.vesoft.nebula.tools.importer.config
+
+/**
+  * SinkCategory
```

Comment:
for example: `SinkCategory` is use to expression the writer's type. Currently support ...